app/services/editions/db/IssueQueries.scala

```diff
@@ -223,7 +223,7 @@ trait IssueQueries {
       .apply()
   }
 
-  def publishIssue(issueId: String, user: User, now: OffsetDateTime) = DB localTx { implicit session =>
+  def publishIssue(issueId: String, user: User, now: OffsetDateTime): String = DB localTx { implicit session =>
     val userName = user.firstName + " " + user.lastName
     val truncatedNow = EditionsDB.truncateDateTime(now)
```

Comment:
we will need as well a function that will do the update of `PublicationEvent ` by updating the `published_on` and `status` columns and return some indication if ti was successful (base on the outcome we decide if we delete the message form SQS)

`published_on` is nullable ?

Reply:
> we will need as well a function that will do the update of `PublicationEvent` by event id, which will update the `published_on` and `status` columns and return some indication if it was successful (base on the outcome, we decide if we delete the message form SQS, now it is boolean)

Agreed. Let's do that in another PR though to avoid this one getting any larger.

> `published_on` is nullable ?

Yes - when we first publish an issue, we write to the events table. As we have not processed a message from SQS at this point, we do not know the `published_on` value (as that represents the time we received the SQS message).

Reply:
👍